### ISSUE TITLE
AWX Operator installer pod crash-loop on Openshift 4.7 crashes during secret_key_configuration.yml

### ISSUE THREAD

**Issue by neilneyman** [ansible/tower-operator#546]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
Attempting to follow the install instructions on openshift, I am able to create the awx-operator pod, but as it runs through the ansible installer scripts, it keeps crashing and restarting during the:
TASK [installer : Check for default secret key configuration] 

##### ENVIRONMENT
* Operator version: 0.13.0
* Kubernetes version: v1.20.0+558d959
* AWX install method: openshift 4.7
##### STEPS TO REPRODUCE

```bash
oc apply -f https://raw.githubusercontent.com/ansible/awx-operator/0.13.0/deploy/awx-operator.yaml
#also needed to manually create the  awx-operator service account and grant roles/permissions as necessary

oc apply -f awx.yml
```

**awx.yml:**
```yaml
apiVersion: awx.ansible.com/v1beta1
kind: AWX
metadata:
  name: awx-demo
spec:
  service_type: nodeport
  ingress_type: none
  hostname: awx-demo-awx-test.apps.myopenshiftdomain.name
 ```

##### EXPECTED RESULTS

AWX Instance should be deployed, including application and postgres pods

##### ACTUAL RESULTS

The AWX-Operator pod is created, and begins running the ansible installer scripts. Watching the logs, it keeps crashing after getting to this point -- below is the last log output before the crash and restart occurs. 
```
TASK [installer : Check for default secret key configuration] ******************
[1;30mtask path: /opt/ansible/roles/installer/tasks/secret_key_configuration.yml:11[0m

-------------------------------------------------------------------------------
{"level":"info","ts":1631715756.4498901,"logger":"logging_event_handler","msg":"[playbook task]","name":"awx-demo","namespace":"awx-test","gvk":"awx.ansible.com/v1beta1, Kind=AWX","event_type":"playbook_on_task_start","job":"2775422040480279449","EventData.Name":"installer : Check for default secret key configuration"}
{"level":"info","ts":1631715765.1688225,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/version","Verb":"get","APIPrefix":"","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.2454638,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/version/openshift","Verb":"get","APIPrefix":"","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.2520235,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis","Verb":"get","APIPrefix":"","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.3431113,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/api/v1","Verb":"get","APIPrefix":"","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.3517456,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/apiregistration.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.3564138,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/apps/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.3651183,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/events.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.369455,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/authentication.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.3735173,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/authorization.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.3779898,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/autoscaling/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4403777,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/batch/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4450235,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/certificates.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.448527,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/networking.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4526231,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/rbac.authorization.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4562054,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/storage.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4595766,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/admissionregistration.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4626076,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/apiextensions.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4660127,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/scheduling.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4704385,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/coordination.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4765074,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/node.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4825592,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/apps.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.4938738,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/authorization.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.5109205,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/build.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.5444665,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/image.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.5558174,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/oauth.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.5938346,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/project.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.6042316,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/quota.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.6133978,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/route.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.622203,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/security.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.6408687,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/template.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.651051,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/user.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.6616066,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/packages.operators.coreos.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.6784272,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/config.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.6849625,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/operator.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.7458613,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/apm.k8s.elastic.co/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.7521973,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/autoscaling.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.756719,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/cloudcredential.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.7609663,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/console.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.766502,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/elasticsearch.k8s.elastic.co/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.8434741,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/imageregistry.operator.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.8482013,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/ingress.operator.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.852459,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/k8s.cni.cncf.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.8566628,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/kibana.k8s.elastic.co/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.8606937,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/loggers.coralogix.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.8652532,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/logging.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.869901,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/machineconfiguration.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.940516,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/monitoring.coreos.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.9469497,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/network.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.951227,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/network.operator.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.9554763,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/networkaddonsoperator.network.kubevirt.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.9596796,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/operators.coreos.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.9641984,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/org.eclipse.che/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715765.9681506,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/samples.operator.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715766.0411983,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/security.internal.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715766.0472317,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/snapshot.storage.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
{"level":"info","ts":1631715766.0520566,"logger":"proxy","msg":"Skipping cache lookup","resource":{"IsResourceRequest":false,"Path":"/apis/tuned.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}

```

##### ADDITIONAL INFORMATION

I've also tried creating a default admin password secret like:
```yaml
---
apiVersion: v1
kind: Secret
metadata:
  name: awx-demo-admin-password
  namespace: awx-test
stringData:
  password: admin
```

**Comment by neilneyman** [comment]
**Update:**

We have resolved this issue by setting this environment variable to the value of the project namespace:
WATCH_NAMESPACE = awx-test

Unfortunately we are having another issue with getting the build to complete.  The PostgreSQL and the application pods are created, but the application container is unable to start.  Using default values, I find this error message in the pod yaml

```
  reason: FailedCreate
      message: >-
        Pod "awx-demo-7bbb564887-q7cnf" is invalid:
        [spec.containers[1].resources.requests: Invalid value: "1": must be less
        than or equal to cpu limit, spec.containers[1].resources.requests:
        Invalid value: "2Gi": must be less than or equal to memory limit,
        spec.containers[2].resources.requests: Invalid value: "500m": must be
        less than or equal to cpu limit, spec.containers[2].resources.requests:
        Invalid value: "1Gi": must be less than or equal to memory limit,
        spec.containers[3].resources.requests: Invalid value: "1Gi": must be
        less than or equal to memory limit,
        spec.containers[3].resources.requests: Invalid value: "500m": must be
        less than or equal to cpu limit]
```

Then I tried patching the requests and memory limits, but it seems no matter what values I specify, the limit requirements are not met
```
  web_resource_requirements:
    requests:
      cpu: 1000m
      memory: 2Gi
    limits:
      cpu: 2000m
      memory: 4Gi
  task_resource_requirements:
    requests:
      cpu: 500m
      memory: 1Gi
    limits:
      cpu: 1000m
      memory: 2Gi
  ee_resource_requirements:
    requests:
      cpu: 500m
      memory: 1Gi
    limits:
      cpu: 1000m
      memory: 2Gi

      reason: FailedCreate
      message: >-
        pods "awx-demo-57d5f44d67-7mrpv" is forbidden: [maximum cpu usage per
        Pod is 2, but limit is 4300m, maximum memory usage per Pod is 1Gi, but
        limit is 8799649792, maximum memory usage per Container is 1Gi, but
        limit is 4Gi, maximum memory usage per Container is 1Gi, but limit is
        2Gi]
```

Or trying to decrease:



```
  web_resource_requirements:
    requests:
      cpu: 500m
      memory: 1Gi
    limits:
      cpu: 1000m
      memory: 1Gi
  task_resource_requirements:
    requests:
      cpu: 500m
      memory: 1Gi
    limits:
      cpu: 1000m
      memory: 1Gi
  ee_resource_requirements:
    requests:
      cpu: 500m
      memory: 1Gi
    limits:
      cpu: 1000m
      memory: 1Gi
      reason: FailedCreate
      message: >-
        pods "awx-demo-5977bc7b4c-zf5d4" is forbidden: [maximum cpu usage per
        Pod is 2, but limit is 3300m, maximum memory usage per Pod is 1Gi, but
        limit is 3430940672]
```

**Comment by neilneyman** [comment]
Wanted to follow-up on this. Looks like we had some resource limits set for the namespace itself that were affecting this, we were able to build it using a completely new project/namespace.    Probably can close this issue.   I did want to note if anyone else is experiencing issues getting this to build on openshift, the additional steps we needed included:

- Manually creating the awx-operator service account and granting the necessary project and cluster-wide roles/permissions
- Setting the WATCH_NAMESPACE  Environment variable on the awx-operator service to the name of the project namespace for the application
- Ensuring enough resources are available to start all of the containers

**Comment by cnukwas** [comment]
@neilneyman I was going to say check LimitRanges settings, good to know that was the case. I installed 0.12 version of the operator and CRDs in a separate namespace in OpenShift 4.7 cluster and don't see any issues other than having to use dynamic storage for storing PostgreSQL data as we were seeing permission errors with NFS share based PV/PVC.
I noticed that by default WATCH_NAMESPACE value is set to "" in the operator YAML, I believe this makes the operator to watch all namespaces rather than the configured namespace. I wasn't aware of this, will need to fix this in our AWX deployments.
